README.md

```diff
@@ -33,6 +35,67 @@ game titles.
 
 At the end, the installation script prints the URL where you can access the demo. Happy gaming!
 
+## Uninstall
+
+In case that you don't want to use the cloud gaming demo any longer you can simply uninstall it from
+your Anbox Cloud Appliance by running the following commands
+
+    cd cloud-gaming-demo
+    sudo ./uninstall.sh
+
+Afterwards all configuration and the cloud gaming demo itself is removed from the machine.
+
+## Secure access via HTTP basic auth
+
+By default the demo is exposed without access control. However you can easily add HTTP basic auth by changing
+the [traefik](https://traefik.io/) configuration at `/var/snap/anbox-cloud-appliance/common/traefik/conf/cloud-gaming-demo.yaml`
+by adding a HTTP basic auth middleware definition (see [here](https://doc.traefik.io/traefik/v2.0/middlewares/basicauth/)
+for more details).
+
+First we have to use the `htpasswd` tool to generate a user/password combination
+
+    apt install -y apache2-utils
+    httpasswd -n <your user name>
+
+On the prompt enter your desired password. The printed user/hashed password combination can now be
+inserted into the traefik configuration. For that open `/var/snap/anbox-cloud-appliance/common/traefik/conf/cloud-gaming-demo.yaml`
+in an editor and change the configuration to look like:
+
+    http:
+    routers:
+        ...
+        middlewares: ["ratelimiter", "strip-demo-prefix", "demo-auth"]
+    middlewares:
+        ...
+        demo-auth:
+            basicAuth:
+                users:
+                - "<user name>:<hashed password>"
+    ...
+
+Afterwards every user will be asked for username and password when accessing the demo site.
+
+## Build
+
+If you want to build the [snap package](https://snapcraft.io) including the demo UI and service yourself
+you first have to install the [snapcraft](https://snapcraft.io) build tool. Please find further details
+[here](https://snapcraft.io/docs/snapcraft-overview)
+
+Once you have snapcraft installed all you have to do is running the following command:
+
+    snapcraft
+
+This will start the build process and procude a `.snap` package at the end which you can install with
```

Comment:
```suggestion
When you have snapcraft installed, run the following command in the root directory of the repository to build the snap:

    snapcraft

This will start the build process and produce a `.snap` package that you can install with the following command:
```

Reply:
I guess you run it in the root folder?